packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -40,7 +40,7 @@ import {
   NodeRuntimeWrapper,
   ResetNodeErrorsKeyProvider,
 } from '@mui/toolpad-core/runtime';
-import { pick } from 'lodash-es';
+import * as _ from 'lodash-es';
```

Comment:
minor thing - just wondering if we are using something to treeshake such imports?
In the past I've used [babel-transform-imports](https://www.npmjs.com/package/babel-plugin-transform-imports) maybe in the future it would be worth to look into that 🤔

Reply:
It is tree shaken by the `swc` minifier in next.js. The import style doesn't influence the behavior.

Reply:
Oh nice, thanks, I didn't know that 👍